### ISSUE TITLE
Close to tray

### ISSUE THREAD

**Issue by inwardmovement** [JosefNemec/Playnite#2135]
Add "close to tray" options, or replace "minimize..." options with "close..." options.

This would allow more flexibility as we could keep the classic minimize (to taskbar) behavior while "closing" to tray. See other applications (Discord/Hangouts for example) behavior.

![_000286](https://user-images.githubusercontent.com/9438102/99975838-fdbe6f80-2da2-11eb-9382-98f661e39dd5.png)

**Comment by JosefNemec** [comment]
I don't understand how existing options are different to what you are proposing.

**Comment by inwardmovement** [comment]
I want the app to be launched "minimized/closed to tray" (just like Discord). Currently it starts minimized in the taskbar.
I cannot set "minimize to tray" because then I lost the ability to minimize (in taskbar). Is it more clear?

**Comment by nickkeane** [comment]
Yeah, having an additional Minimize option to have Playnite start and occupy the tray instead of the taskbar would be a great addition.

**Comment by Crwmy** [comment]
Adding my vote for this.

**Comment by baterflyrity** [comment]
+1
Please, add this option.

**Comment by solo-nko** [comment]
Would also love to see this feature.

**Comment by JosefNemec** [comment]
Was added in Playnite 10.
